### ISSUE TITLE
Is pre-trained VQGAN essential

### ISSUE THREAD

**Issue by Whiteglad** [xuekt98/BBDM#9]
Hello, I reproduced your method in our own dataset, but the loss didn't decrease.  Is pre-trained VQGAN necessary for the training of specific datasets？

**Comment by xuekt98** [comment]
Thank you for your attention. VQGAN is necessary only if you want to process in latent sapce.
